mbbsd/bbs.c

```diff
@@ -3690,6 +3711,32 @@ mark_post(int ent, fileheader_t * fhdr, const char *direct)
     return change_post_mode(ent, fhdr, direct, FILE_MARKED);
 }
 
+static int
+mark_suspicious(int ent, fileheader_t * fhdr, const char *direct)
+{
```

Comment:
People may accidentally hit `M` when they try to `m` the post if they have their caps lock on.

I think we should just implement this in `{ 1, manage_post }, // Ctrl('E')`.

Reply:
Thanks. ```mark_suspicious``` has been moved into ```manage_post```.